Code/skyrim_ui/src/app/components/notification-popup/notification-popup.component.html

```diff
@@ -1,5 +1,10 @@
-<div class="notification-popup" (click)="clickNotification()" (mouseenter)="onEnter()" (mouseleave)="onLeave()">
-  <div *ngIf="notification.player?.avatar" class="friendavatar" [ngStyle]="{ 'background-image': 'url(' + notification.player.avatar + ')' }"></div>
-  <div class="text-connection-popup">{{ notification.message }}</div>
-  <div *ngIf="isConnected" class="friendstatus" [class.online]="notification.player?.online"></div>
+<div *transloco="let t" class="notification-popup" (click)="clickNotification()" (mouseenter)="onEnter()" (mouseleave)="onLeave()">
+  <img *ngIf="notification.imageUrl" src="{{notification.imageUrl}}" />
+  <div *ngIf="notification.icon" class="icon">
+    <fa-icon  [icon]="notification.icon"></fa-icon>
+  </div>
+  <p>{{ t(notification.messageKey, notification.messageParams) }}</p>
+  <button *ngFor="let action of notification.actions" (click)="action.callback()">
+    {{t(action.nameKey)}}
```

Comment:
Please add whitespace around interpolation content for consistency

Reply:
fixed